### PR TITLE
Supplement missing entity templates

### PR DESCRIPTION
**Pull request by LaszloKecskes**
fixes #

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

**Review by fnocetti — CHANGES_REQUESTED** [review]
The code looks good, but I think it's missing the translations context. I will proceed to the manual testing.

**Comment by LaszloKecskes** [comment]
> ...but I think it's missing the translations context. 

Indeed, it does. Nice catch.

File: app/api/migrations/migrations/162-supplement-missing-entity-templates/specs/162-supplement-missing-entity-templates.spec.ts [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+            _id: expect.any(ObjectId),
+            label: 'Date modified',
+            name: 'editDate',
+            isCommonProperty: true,
+            type: 'date',
+            prioritySorting: false,
+          },
+        ],
+        properties: [],
+        color: '##ff0000',
```
Comment (fnocetti) [inline app/api/migrations/migrations/162-supplement-missing-entity-templates/specs/162-supplement-missing-entity-templates.spec.ts]: The hex color should be prefixed with 1 single #

**Review by fnocetti — CHANGES_REQUESTED** [review]
It works as expected. Left one comment that needs addressing.

File: app/api/migrations/migrations/162-supplement-missing-entity-templates/specs/162-supplement-missing-entity-templates.spec.ts (hunk 1 above)
Comment (LaszloKecskes) [reply app/api/migrations/migrations/162-supplement-missing-entity-templates/specs/162-supplement-missing-entity-templates.spec.ts]: Corrected.

**Review by fnocetti — APPROVED** [review]
LGTM!

**Pull request merged**